docs/content/gs-writing-walkthroughs-proc.adoc

```diff
@@ -84,7 +83,8 @@ cd walkthrough-template
 Make sure you have a directory named `walkthrough-template` on your system containing a `walkthroughs` subdirectory.
 
 [type=verificationFail]
-Try downloading the link:https://github.com/integr8ly/walkthrough-template/archive/master.zip[zip] file and creating a repo.
+Try downloading the link:https://github.com/integr8ly/walkthrough-template/archive/master.zip[zip] file and creating a repository
```

Comment:
let's try to avoid 'Try'
(altho, if you check i think you'll find this whole section is commented out)

Reply:
Have changed "Try downloading the" to "Download the".  It is indeed commented out but may as well fix it anyways.